core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -249,20 +249,27 @@ object ConsumerGroupCommand extends Logging {
       for ((groupId, (state, assignments)) <- members) {
         if (shouldPrintMemberState(groupId, state, size(assignments))) {
           // find proper columns width
-          var (maxGroupLen, maxConsumerIdLen, maxHostLen, maxClientIdLen) = (15, 15, 15, 15)
+          var (maxGroupLen, maxConsumerIdLen, maxGroupInstanceIdLen, maxHostLen, maxClientIdLen, nullGroupInstanceId) = (15, 15, 17, 15, 15, true)
           assignments match {
             case None => // do nothing
             case Some(memberAssignments) =>
               memberAssignments.foreach { memberAssignment =>
                 maxGroupLen = Math.max(maxGroupLen, memberAssignment.group.length)
                 maxConsumerIdLen = Math.max(maxConsumerIdLen, memberAssignment.consumerId.length)
+                maxGroupInstanceIdLen =  Math.max(maxGroupInstanceIdLen, memberAssignment.groupInstanceId.length)
                 maxHostLen = Math.max(maxHostLen, memberAssignment.host.length)
                 maxClientIdLen = Math.max(maxClientIdLen, memberAssignment.clientId.length)
+                nullGroupInstanceId = nullGroupInstanceId && memberAssignment.groupInstanceId.length == 0
```

Comment:
nit: it's a little surprising to see `nullGroupInstanceId` and `groupInstanceId.length` in the same expression. I'd expect that to be an NPE if I didn't see how it is caused. I think I'd suggest inverting this. Instead of `nullGroupInstanceId`, how about `includeGroupInstanceId`? Then this becomes
```scala
includeGroupInstanceId = includeGroupInstanceId || memberAssignment.groupInstanceId.length > 0
```

Reply:
Hi @hachikuji , thank you for your suggestion, it's very useful to me, done!